.circleci/config.yml

```diff
@@ -11,11 +12,25 @@ jobs:
       - node/install-packages
       - run:
           command: npm run build
+      - run:
```

Comment:
if you are only going to use the "command" parameter for the `run` step, you can simplify these to:

`- run: npm run lint`